src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -852,6 +891,25 @@ def add_agent_manager(self, group=None, config_file=None, config_temp=None):
                 self.agent_manager_class)
         )
 
+    def get_server_manager(self, group=None):
```

Comment:
I don't think this is needed.  If a test starts multiple server groups, like `pool/destroy_tests.py`, it should internally manage which `self.server_managers` index is used for each server group.

Reply:
Removed